runtime-library/utils/object_utils.c

```diff
@@ -412,9 +416,12 @@ call_wasm_func_with_boxing(wasm_exec_env_t exec_env, dyn_ctx_t ctx,
     }
 
     /* reserve space for the biggest slots */
-    bh_memcpy_s(argv, bsize - occupied_slots, &(context.gc_obj),
+    bh_memcpy_s(argv, sizeof(wasm_anyref_obj_t), &(context.gc_obj),
                 sizeof(wasm_anyref_obj_t));
     occupied_slots += sizeof(wasm_anyref_obj_t) / sizeof(uint32);
+    bh_memcpy_s(argv + occupied_slots, sizeof(wasm_anyref_obj_t),
```

Comment:
The second parameter is "remaining space from dest address", not the size required to store the value.
e.g. your `argv` is 10 bytes long, and here if `occupied_slots` is 9, then the second parameter should be 1, then the `bh_memcpy_s` knows you don't have enough space to store the value and raise error. If you use `sizeof(wasm_anyref_obj_t)` as the second parameter, you escape the check and may cause serious errors.

Reply:
Get it, I will fix it